context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -49,9 +53,40 @@ function SearchWrapper(props) {
     };
   }, [searchkit, setSearchHitsCount]);
 
+  const [message, setMessage] = useState(null);
+  const { elasticsearchEndpoint, nexusToken } = useContext(AppContext);
+  searchkit.setQueryProcessor((query) => {
```

Comment:
It seems searchkit has a `QueryAccessor` is this more along the lines of what we should use?

Reply:
Will look into that. Thanks!